src/Kernel32.Shared/Kernel32.cs

```diff
@@ -208,11 +208,24 @@ public enum FormatMessageFlags
         /// <param name="hObject">A valid handle to an open object.</param>
         /// <returns>
         /// If the function succeeds, the return value is nonzero.
-        /// If the function fails, the return value is zero.To get extended error information, call GetLastError.
+        /// <para>If the function fails, the return value is zero.To get extended error information, call GetLastError.</para>
         /// </returns>
         [DllImport(nameof(Kernel32), SetLastError = true)]
         public static extern bool CloseHandle(IntPtr hObject);
 
+        /// <summary>
+        /// Retrieves the thread identifier of the calling thread.
+        /// </summary>
+        /// <returns>The thread identifier of the calling thread.</returns>
+        [DllImport(nameof(Kernel32))]
+        public static extern uint GetCurrentThreadId();
```

Comment:
I'm more for keeping to the values found in the original headers as interpreting this value as a two complement signed value is clearly not what was intended by them.

I suspect that the BCL kept with `int` everywhere instead of `uint` due to CLS complicance, and I don't think that we should care about that here.